algorithms/glued_trees/glued_trees.ipynb

```diff
@@ -0,0 +1,438 @@
+{
```

Comment:
<p>The algorithm and speedup are in the setting of oracular queries. Needs to be mentioned </p><p>There is an older paper that first set this problem - <a href="http://arxiv.org/abs/quant-ph/0209131" target="_blank">arXiv:quant-ph/0209131</a>, needs to be mentioned.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/151/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>